vocabularies/gsq-roles.ttl

```diff
@@ -150,6 +152,34 @@ isorole:publisher a skos:Concept ;
     skos:prefLabel "Publisher"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/gsq-roles>.
         
+gsqr:data-creator
+  a skos:Concept ;
+  a dcat:Role ;
+  rdfs:label "Data Creator"@en ;
+  skos:closeMatch <http://linked.data.gov.au/def/iso19115-1/RoleCode/author> ;
+  skos:definition "Responsible for creating and updating data, products and metadata and storing in an appropriate system."@en ;
+  skos:prefLabel "Data Creator"@en ;
+  skos:inScheme <http://linked.data.gov.au/def/gsq-roles> ;
+  skos:topConceptOf <http://linked.data.gov.au/def/gsq-roles> .
+
+gsqr:data-librarian
+  a skos:Concept ;
+  a dcat:Role ;
+  rdfs:label "Data Librarian"@en ;
+  skos:broader gsqr:manager ;
```

Comment:
@DavidCrosswellGSQ We are using the terms Librarian, Steward and Administrator as specialisations of the QGEA Information Management term "Data Manager". A visual representation of how we are using the terms is attached.

We could look using the skos:related term and providing a description that is it a specialisation of Data Manager.

![image](https://user-images.githubusercontent.com/56572575/101296696-90194580-3870-11eb-83d6-865f2cedbf06.png)

Reply:
@broughd-qld - skos:related is a good way to handle this. The relationship is more "is a" than rather than a hierarchical relationshp.
Please create the skos:related one-way from the Steward/Librarian to the Data Manager.

Reply:
done